app/schema.graphql

```diff
@@ -1722,6 +1890,11 @@ enum ScalarDriftMetric {
   jsDistance
 }
 
+enum ScoreDirection {
+  MINIMIZE
+  MAXIMIZE
```

Comment:
these constants are hard for me to understand - I'm guessing maximize means "higher is good" but as a first time reader this was pretty confusing.

Reply:
Alternatively this could be 

OptimizationDirection { MAXIMIZE
MINIMIZE
}